common/wuanalysis/anawu.cpp

```diff
@@ -95,15 +95,89 @@ class WuScope : public CInterface, implements IWuEdge, implements IWuActivity
 };
 
 //-----------------------------------------------------------------------------------------------------------
+typedef enum { WuOptionType_MSec, WuOptionType_Seconds, WuOptionType_Percent } WuOptionType;
+
+struct WuOption
+{
+    const char * name;
+    __int64 value;
+    WuOptionType type;
+};
+
+constexpr struct WuOption wuOptionsDefaults[]
+= { { "minInterestingTime", 10, WuOptionType_MSec},
```

Comment:
there isn't any checking that the options are in the same order as the enumeration, or that they have the same number of elemens.  It would be better if there was.